recipe/meta.yaml

```diff
@@ -20,12 +19,12 @@ requirements:
     - python
     - pip
     - cython
+    - numpy >=1.16
```

Comment:
You can remove the `>-1.16` and let our pinning use the min supported version, which is higher than 1.16. Having it there will end up building with the latest and only running with the latest.

Reply:
okay, done